CHANGELOG.md

```diff
@@ -58,8 +58,8 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### New
 
-- TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
 - **General**: Declarative parsing of scaler config ([#5037](https://github.com/kedacore/keda/issues/5037)|[#5797](https://github.com/kedacore/keda/issues/5797))
+- **CloudEventSource**: Introduce ClusterCloudEventSource ([#3533](https://github.com/kedacore/keda/issues/3533))
```

Comment:
@JorTurFer I don't like the current prefix, do we have list of the ones supported again? This feels like an operations or so category
```suggestion
- **Operations**: Introduce `ClusterCloudEventSource` for cluster-wide event subscription ([#3533](https://github.com/kedacore/keda/issues/3533))
```